### DISCUSSION TITLE
Force update of add-on?

### DISCUSSION THREAD

**Discussion by RutgerDiehard** [ESPresense/ESPresense-companion#358] [General]
Now that 0.4.0 has been released, is there any way to force an update of the companion add-on? I've tried checking for updates in both settings and add-ons, restarting the add-on, unset and reset the "Auto Update" switch for the add-on. But the companion is still firmly sat on 0.3.40. Is it really a remove and reinstall?

**Comment by DTTerastar** [comment]
No, I didn't push out 0.4.0 because of bugs.  It's out now.  I am working on one more bug, 0.4.1 will be the bees knees.
